### DISCUSSION TITLE
Bubble.io API

### DISCUSSION THREAD

**Discussion by dwaynesenpai25** [community/community#121602] [Apps, API and Webhooks]
### Select Topic Area

Question

### Body

How can get that work

**Comment by samus-aran** [comment]
Hi @dwaynesenpai25 ,

Thanks for being a part of the GitHub Community, we're glad you're here!

If you're looking for help for this specific topic, you might want to try asking for help somewhere that focuses on this project, such as the [project's community](https://forum.bubble.io/). It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!
